alembic/README.md

```diff
@@ -8,16 +8,17 @@ normally do.
 commands:
 ```
 sep drop_uber_db
-sep alembic upgrade head
+sep alembic upgrade heads
+# If you have only the uber plugin installed, use `alembic upgrade head` instead of `heads`
```

Comment:
I'm pretty sure we can remove this line. `sep alembic upgrade heads` (plural heads) should always work even if there is only one head (from just one plugin).